articles/tech-resources/a-b-testing-sparkpost.md

```diff
@@ -9,31 +9,42 @@ An A/B test, in summary, is a common method of comparing templates to see which
 
 **What benefits do I get from using A/B testing?** 
 
-An A/B test is a method of comparing templates to see which one performs better by splitting a given recipient list across the number of templates you are comparing. Performing these tests on a limited percentage of a recipient list will allow you to send to the winning template your remaining audience, which can help drive better engagement results for your campaigns.
+An A/B test is a method of comparing 2 or more templates to see which one performs better. That is, which one provides a higher unique open or click rate. The SparkPost A/B Testing API allows users to compare multiple templates in the context of transactional or app generated messages, rather than lists, to see which one performs better for a set number of recipients (Sample Size) or over a set period of time. 
+
+Some studies have shown that doing A/B testing on email content (vs. not doing any testing) can lift open rates by 13% or more. So here's a step by step guide to A/B Testing at SparkPost to get your started. Note that enhancements are coming over the months ahead (we're agile too!) - so please give us feedback on what you would like to see improved in our [Beta Program](https://www.sparkpost.com/sparkpost-beta-tester/).
 
 **Creation and review of A/B testing through the API** 
 
-Please refer to the [A/B testing API documentation](https://developers.sparkpost.com/api/ab-testing.html) for specifics on how to create and retrieve A/B tests.
+Please refer to the [A/B testing API documentation](https://developers.sparkpost.com/api/ab-testing.html) for specifics on how to create and retrieve A/B test results. As a basic flow:
+  1.  Call the A/B testing API to create/edit your A/B Test. 
+  2.  Once you have a A/B Test ID, use it use it in the Transmissions API instead of your regular content jSON. The A/B Testing functionality will then take over and send either the default template you've defined as part of your A/B Test set up process or one of the variants if you're running an active.
```

Comment:
`content JSON`

Reply:
maybe something like:

The A/B Testing functionality will then take over and send either the default template or one of the variants you've defined as part of your A/B Test set up process, once the test is active.

thoughts?  `running an active` definitely was missing something